mkt/comm/tests/test_views.py

```diff
@@ -448,10 +448,37 @@ def test_cors_allowed(self):
         self.assertCORS(res, 'get', 'post', 'patch')
 
 
+class TestNoteListView(RestOAuth):
+
+    def setUp(self):
+        super(TestNoteListView, self).setUp()
+        self.extension = extension_factory()
+        self.extension_author = user_factory(email='addon@author.com')
+        self.extension_thread = self.extension.threads.create(
+            _extension_version=self.extension.latest_version)
+        self.extension_note = self.extension_thread.notes.create(
+            author=self.extension_author, body='extension note')
+
+        self.app = app_factory()
+        self.app_author = user_factory(email='app@author.com')
+        self.app_thread = self.app.threads.create(
+            _version=self.app.current_version)
+        self.app_note = self.app_thread.notes.create(
+            author=self.app_author, body='webapp note')
+
+        self.url = reverse('api-v2:comm-note-list-all')
+
+    def test_list(self):
+        res = self.client.get(self.url)
```

Comment:
@diox i'm a bit stuck here. when i hit the endpoint, i get:

```
404
/api/v2/comm/notes/
{"detail": "Not found"}
```

The URL is pointing to the right view, but the view doesn't seem to be getting called. Looks like it's hitting some detail endpoint.